gossip/src/cluster_info_metrics.rs

```diff
@@ -113,6 +113,10 @@ pub struct GossipStats {
     pub(crate) gossip_pull_request_verify_fail: Counter,
     pub(crate) gossip_pull_response_verify_fail: Counter,
     pub(crate) gossip_push_msg_verify_fail: Counter,
+    pub(crate) gossip_transmit_loop_time: Counter,
+    pub(crate) gossip_transmit_loop_itrs_since_last_report: Counter,
```

Comment:
nit: `itrs` -> `iterations` we usually spell out all words without abbreviations